flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -227,6 +227,14 @@
 	public static final String TASK_MANAGER_NETWORK_NUM_BUFFERS_KEY = "taskmanager.network.numberOfBuffers";
 
 	/**
+	 * Boolean flag to enable/disable more detailed metrics about inbound/outbound network queue lengths
+	 */
+	@PublicEvolving
+	public static final ConfigOption<Boolean> NETWORK_DETAILED_METRICS_KEY =
```

Comment:
actually, while looking into that, I noticed that some similar options already went into `TaskManagerOptions` but are also prefixed with `taskmanager.net.` so I'll drop the new config there and revert to this namespace